packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -503,6 +503,43 @@ describe('<Autocomplete />', () => {
       });
     });
 
+    it('should apply the multiple class', () => {
+      const { container } = render(
+        <Autocomplete
+          openOnFocus
+          options={[]}
+          renderInput={(params) => <TextField {...params} />}
+          multiple
+        />,
+      );
+      expect(container.querySelector(`.${classes.inputRoot}`)).to.have.class(classes.multiple);
+    });
+
+    it('should allow to override the style for the multiple class', () => {
+      const theme = createTheme({
+        components: {
+          MuiAutocomplete: {
+            styleOverrides: {
+              root: {
+                [`& .${classes.inputRoot}.${classes.multiple}`]: {
+                  mixBlendMode: 'darken',
+                },
+              },
```

Comment:
We want to test using `multiple` class:
```suggestion
              multiple: {
                  mixBlendMode: 'darken',
              },
```